tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -1450,6 +1509,35 @@ port::Status CudnnSupport::DoRnnForwardImpl(
   }
 
   if (!is_training) {
+#if CUDNN_VERSION >= 7201
+    if (input_desc.is_var_seq_lens()) {
```

Comment:
how about making `input_desc.is_var_seq_lengths` return false by default, and
```
if (var_seq_len) {
# if cudnn_version > 7201
  call XXXEx()
# else
  raise error
# endif
} else {
  call the non-ex version.
}
```

Same for training methods.